small_grants.md

```diff
@@ -0,0 +1,153 @@
+competitive atmosphere, **candidates must declare to the committee interest 
+before solving the chosen issue** and, upon approval by the selection committee, 
+are given an exclusive time  interval (defaulting to one month) to solve the issue. 
+Payout is done upon completion of the chosen project and acceptance by the steering
+council.
+
+## Donating to the Program
+
+If you wish to donate to the SciML Small Grants Program, 
+[please donate via NumFOCUS](https://numfocus.org/donate-to-sciml). SciML via NumFOCUS
+is a 501(c)(3) public charity in the United States. Your donation is tax-deductible 
+to the extent provided by US law. General donated funds are used for the developer
+programs such as the small grants program and the SciML Fellowship.
+
+Donations can be earmarked towards specific projects. In order for an earmarked
+project to become a part of part of the SciML Small Grants program it must be
+approved by the SciML Steering Council. The reason why all chosen projects must be
+vetted by the Steering Council is that we want to be fair to the potential contributors,
+and this means we must ensure that the suggested projects will have a prompt review
+process and proper maintanance beyond the timeframe of the project. For this reason,
+we suggest discussing with maintainers in the official Slack/Zulip before making
+an earmarked donation.
+
+## List of Current Projects
+
+### Fix and Update the "Simple Handwritten PDEs as ODEs" Benchmark Set ($200)
+
+The "Simple Handwritten PDEs as ODEs" benchmarks have been failing for awhile.
+They need to be updated to the "new" linear solve syntax introduced in 2022.
+When updated, these benchmarks should serve as a canonical development
+point for PDE-specific methods, such as implicit-explicit (IMEX) and
+exponential integrators.
+
+**Information to Get Started**: The 
+[Contributing Section of the SciMLBenchmarks README](https://github.com/SciML/SciMLBenchmarks.jl?tab=readme-ov-file#contributing)
+describes how to contribute to the benchmarks. The benchmark results are
+generated using the benchmark server. Half of the benchmarks are setup
+using hand-discretized finite difference stencils for the PDE, the other
+half use ApproxFun.jl in order to do a pseudospectral discretization.
+A direct pseudospectral discretization via manual FFTs and operator
+construction would also be fine.
+
+**Related Issues**: https://github.com/SciML/SciMLBenchmarks.jl/issues/929
+
+**Success Criteria**: Pull requests which update the benchmarks in the
+folder https://github.com/SciML/SciMLBenchmarks.jl/tree/master/benchmarks/SimpleHandwrittenPDE
+to be sucessful with current Julia and package version (v1.10) without
+erroring, generating work-precision diagrams. In addition, these should be updated
+to give a more clear definition of the PDE being solve, adding a LaTeX
+description of the equations to the top of the file. 
+
+**Reviewers**: Chris Rackauckas
```

Comment:
What amount of commitment would a reviewer be expected to put in, and what level of support from the reviewer/SciML team would a potential contributor expect?

Additionally, would you want to have an introductory call to clarify what exactly you're expecting, once a contributor is accepted?  Should that be a more formalized process?